### PR TITLE
Antag spawner items and blob cores now poll candidates instead of grabbing candidates with no poll

### LINKED ISSUE #20593: Syndicate operative reinforcements, syndicate cyborgs, blobs grab ghosts without popop
Given that I introduced "WOULD YOU LIKE TO PLAY AS A GIANT SPIDER" messages for all other ghost roles, we might as well go the whole hog and finish them off.

Because nothings more annoying than observing idly while talking in #coderbus before suddenly being dragged into existence as a syndiborg.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #20593

### CODE REVIEW

File: code/game/gamemodes/blob/blobs/core.dm [hunk 1]
```diff
@@ -63,7 +63,8 @@
 
 /obj/effect/blob/core/Life()
 	if(!overmind)
-		create_overmind()
+		if(!qdeleted(src))
+			addtimer(src, "create_overmind", 0)
```
Comment (optimumtact) [inline code/game/gamemodes/blob/blobs/core.dm]: What the fuck is this shit

File: code/game/gamemodes/blob/blobs/core.dm [hunk 2]
```diff
@@ -63,7 +63,8 @@
 
 /obj/effect/blob/core/Life()
 	if(!overmind)
-		create_overmind()
+		if(!qdeleted(src))
```
Comment (optimumtact) [inline code/game/gamemodes/blob/blobs/core.dm]: why would it be qdeleted here

**Review by optimumtact — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/blob/blobs/core.dm (hunk 2 above)
Comment (ChangelingRain) [inline code/game/gamemodes/blob/blobs/core.dm]: I'm hoping wildly that this fixes an unreported bug where the core fucking makes an overmind in nullspace(obviously bad!)

File: code/game/gamemodes/blob/blobs/core.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/blob/blobs/core.dm]: It's probably not needed; does process care about sleeps in it, anymore?

Comment (optimumtact) [inline code/game/gamemodes/blob/blobs/core.dm]: it shouldn't

File: code/game/gamemodes/blob/blobs/core.dm (hunk 2 above)
Comment (optimumtact) [inline code/game/gamemodes/blob/blobs/core.dm]: it's morelikely a blob being spawned in nullspace in the firstplace

**Comment by KingPhilipIII** [comment]
Oh. Make sure the blob core spawns when the poll is complete. On Paradise the core spawns immediately and the blob is totally vulnerable for thirty seconds to get a player dropped in. It really fucks em over sometimes.

**Comment by ChangelingRain** [comment]
The blob event already waits for players, this is just for the actual blob core itself, and the poll lasts 5 seconds.

**Comment by KingPhilipIII** [comment]
Right.

File: code/game/gamemodes/blob/blobs/core.dm (hunk 2 above)
Comment (ChangelingRain) [inline code/game/gamemodes/blob/blobs/core.dm]: The only times I've seen it happen is when a blob core is killed and deletes its overmind sooooo

**Review by optimumtact — APPROVED** [review]

**Pull request merged**